docs-source/filter.yml

```diff
@@ -0,0 +1,3 @@
+apiRules:
+  - exclude:
+      uidRegex: "^Auth0.OidcClient.Resource"
```

Comment:
There where a lot of Resources being added to the docs, I filtered them so that they get removed from the docs.

Here is an example in the current docs: https://auth0.github.io/auth0-oidc-client-net/api/Auth0.OidcClient.Resource.String.html

Running the docs now adds about 10 of this (Resource.Color, Resource.Drawable, ...), so filtering those ensures they are not included.